timberware/cinesync
# closes #13: Implement caching service

## Issue

closes #13 

## Type of Change



## Description
This  is a first implementation of caching for lists and movies. In a follow-up PR, it can be abstracted to its own service, and it'll include caching users.

## Testing the PR
- Make sure there's a way to check the content of a redis db.
- Start the redis db using the new changes in `docker-compose.yml`.
- Assert that fetching movies before and after using cache return correct responses.

## Checklist
